### ISSUE TITLE
Error when loading EQTransformer "original" model

### ISSUE THREAD

**Issue by tmgreenfield1101** [seisbench/seisbench#193]
When loading the "original" EQtransformer model as:
eqt_model = sbm.EQTransformer.from_pretrained("original")

I get the error as below. I'm not an expert, but does this suggest the format of the EQTransformer original model has changed and seisbench needs a small update to reflect this.

RuntimeError: Error(s) in loading state_dict for EQTransformer:
	Missing key(s) in state_dict: "pick_lstms.0.weight_ih_l0", "pick_lstms.0.weight_hh_l0", "pick_lstms.0.bias_ih_l0", "pick_lstms.0.bias_hh_l0", "pick_lstms.1.weight_ih_l0", "pick_lstms.1.weight_hh_l0", "pick_lstms.1.bias_ih_l0", "pick_lstms.1.bias_hh_l0". 
	Unexpected key(s) in state_dict: "pick_lstms.0.cell_f.weight_ih", "pick_lstms.0.cell_f.weight_hh", "pick_lstms.0.cell_f.bias_ih", "pick_lstms.0.cell_f.bias_hh", "pick_lstms.1.cell_f.weight_ih", "pick_lstms.1.cell_f.weight_hh", "pick_lstms.1.cell_f.bias_ih", "pick_lstms.1.cell_f.bias_hh".

**Comment by tmgreenfield1101** [comment]
Thanks for the quick reply. I'm using python=3.8 and pytorch='2.0.0+cu117'. I cloned the seisbench repositiry and after a little faff installed it using python ./setup.py install. Something must have gone wrong with the installation as the version number quoted is 0.0.0, which when I initially tried to use seisbench meant I've had to fudge it to 1.0.0 in order for some features to work.

The suggestion to redownload the local cache worked perfectly. I had an old version (v2) stored there. Everything is working since I've deleted those files and seisbench has redownloaded them.
